### PR TITLE
Switch back to horrible legacy multiline strings

### PR DESCRIPTION
**Pull request by thomaswilburn**

### CODE REVIEW

File: test/can-view-parser-test.js [hunk 1]
```diff
@@ -852,7 +838,9 @@ testHelpers.dev.devOnlyTest('Fix false warning on missing closed quote (canjs/ca
 		equal(count, teardown());
 	};
 
+	/* jshint ignore:start */
```
Comment (phillipskevin) [inline test/can-view-parser-test.js]: why is this needed?

Comment (cherifGsoul) [reply test/can-view-parser-test.js]: Because jshint complains from unused variable

Comment (phillipskevin) [reply test/can-view-parser-test.js]: Why is the variable needed I mean.

Comment (cherifGsoul) [reply test/can-view-parser-test.js]: Is used in stache template
